axelor-base/src/main/resources/domains/BaseBatch.xml

```diff
@@ -1,21 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<domain-models xmlns="http://axelor.com/xml/ns/domain-models"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://axelor.com/xml/ns/domain-models http://axelor.com/xml/ns/domain-models/domain-models_5.0.xsd">
-
-  <module name="base" package="com.axelor.apps.base.db"/>
-
-  <entity name="BaseBatch" lang="java">
```

Comment:
Why remove this object ?

Reply:
Changes were made, now we don't remove the object, the undefined method actionTarget is now defined, code is not required.